docs/guide/pieces.md

```diff
@@ -1,396 +1,85 @@
----
-title: "Pieces"
----
-
 # Pieces
-What is a Piece?
 
-As experienced A2 developers know, a piece type module defines structure and behavior for a single type of content, such as blog posts, products, or events. If you have hundreds of documents of the same type, that's a piece. If you want to let the user view them in a certain sorting order, that's a piece. If the same item might be appropriate to display on many different pages, that's a piece. Blog posts, events and products are all excellent examples. Less obvious examples include "category" or "tag" content types that serve to create a taxonomy for other pieces.
+"Piece types" are classifications for standalone, structured content. Each database document created with a piece type module is known as a "piece". The piece type defines how pieces are structured, how end users interact with them, and how they might interact with other content. Blog posts, events, and people profiles are all common examples of piece types.
+
+Use cases for pieces include:
+- Content that will be listed using pagination, such as **articles in a blog**
+- Content of a particular type that will help to organize other content, such as **teams that connect to people pieces**
+- Content that will be selectively displayed based on end user input, such as **products on a store's website**
 
-Apostrophe provides both pieces and "[piece pages](piece-pages.md)," which afford your users a way to browse, filter, paginate and discover pieces of a particular kind. For more information about using piece pages, see the [piece pages section](piece-pages.md).
+Pieces are distinct from "pages" in Apostrophe in that they do not inherently have an internal hierarchy: generally speaking, one piece is not the "parent" of another of the same type. Pieces also do not necessarily have dedicated web pages for each one. In the "teams" and "people" example, there may be a web page for each team, but the people are only displayed within those team pages -- not on their own pages.
```

Comment:
```suggestion
Pieces do not always have a dedicated slug or web page. For example, in the "author" and "article" example, there will likely be a page for every article, but not a page for every author. 
```

I dont really understand what 'internal hierarchy' means here, or if its necessary to call attention to the fact that they are orphan documents. To me, this implies that if you deleted a piece-type, everything would still be fine, but thats not the case.

Reply:
It refers to the page tree and pieces not having inherent relationships with one another the way pages do.